assets/NonFunctionalRequirements

```diff
@@ -0,0 +1,8 @@
+Kwaliteitseisen
+
+Non-functionele eisen:
+1. Gebruikersgegevens moeten beveiligd zijn
```

Comment:
Dit is iets te vaag. Wil je het achter slot en grendel, bewaakt door een gewapende militie?

Reply:
Wat wij uiteindelijk gaan doen is het beveiligen met encryptie en JWT (JSON Web Token).